posthog/api/capture.py

```diff
@@ -289,6 +296,23 @@ def get_event(request):
                 ),
             )
 
+    for future in futures:
+        try:
+            future.get(timeout=10)
+        except KafkaError:
+            # TODO: distinguish between retriable errors and non-retriable
+            # errors, and set Retry-After header accordingly.
+            return cors_response(
+                request,
+                generate_exception_response(
+                    "capture",
+                    "Unable to store event. Please try again. If you are the owner of this app you can check the logs for further details.",
```

Comment:
I guess events rather than event? Unfortunately it's possible that some go through and some don't but 🤷‍♂️

Reply:
Yeah we could give some feedback here on which messages in the batch failed, maybe worth it.

Reply:
nice to have, but not a blocker